aws/asg-ebs/README.md

```diff
@@ -36,6 +36,7 @@ an auto scaling group that uses the template.  An EBS file system is mounted.
  - `ebs_mkfs_extraopts` - Extra options to pass to the mkfs command. Default: ""
  - `ebs_fs_type` - Type of filesystem to create. Default: `ext4`
  - `ebs_mountopts` - Mount options to include in /etc/fstab, default is "defaults,noatime"
+ - `tags` - A list of tag definitions in JSON format to be applied to the asg.
```

Comment:
Is it really "JSON format" if it doesn't have commas? (Same question for the other one further down.)

Reply:
There's a comma on line 74.  ;-)  I just copied the description from the asg module's README.md.